### ISSUE TITLE
regex support in additional_files_or_dirs:

### ISSUE THREAD

**Issue by NasrHassanein** [metacloud/molecule#1060]
# Issue Type

- Feature request

# Molecule and Ansible details

```
ansible, version 2.4.2.0
molecule, version 2.5.0
```

- Molecule installation method:  pip
- Ansible installation method:  pip

# Desired Behaviour

[additional_files_or_dirs](https://molecule.readthedocs.io/en/latest/configuration.html#testinfra) is really good. I think it would be better if you added wildcard and regex support. That will enable us to include the tests inside the ansible role itself.

so it could be something like the following:
```
verifier:
  name: testinfra
  additional_files_or_dirs:
    - ../../ansible-roles/test_*.py
    - ../../ansible-roles/*/tests/
```

**Comment by retr0h** [comment]
@dj-wasabi  interested in implementing this one?

**Comment by dj-wasabi** [comment]
@retr0h I was already thinking about it. I'm not able to do anything after the weekend though. If issue is still open after this weekend, I'll give it a try.

**Comment by hmarlo** [comment]
Nice feature! I definitely will use this.  Now we will be able to have the tests grouped in one folder:

```
tests/
    test_role_X.py
    test_role_Y.py
roles/
    role_X/
    role_Y/
```

I commented the WIP merge request, thanks @dj-wasabi

**Comment by retr0h** [comment]
Implemented in 2.8.
